docs/Installation/topics/adding_NFS_storage.rst

```diff
@@ -0,0 +1,275 @@
++------------------------------------+---------------------------------------+------------------------------+
+| ``persistentVolume.accessModes``   | Persistent volume access modes        | ``ReadWriteMany``            |
++------------------------------------+---------------------------------------+------------------------------+
+| ``persistentVolume.existingClaim`` | Name of existing claim                | ``""``                       |
++------------------------------------+---------------------------------------+------------------------------+
+| ``persistentVolume.storageClass``  | Persistent volume storage class       | ``cosmos``                   |
++------------------------------------+---------------------------------------+------------------------------+
+| ``persistentVolume.size``          | Persistent volume size                | ``1Gi``                      |
++------------------------------------+---------------------------------------+------------------------------+
+| ``persistentVolume.readOnly``      | If true, persistent volume will be    | ``false``                    |
+|                                    | read-only                             |                              |
++------------------------------------+---------------------------------------+------------------------------+
+
+Specify each parameter using the ``--set key=value[,key=value]``
+argument to ``helm install``. For example,
+
+.. code:: bash
+
+    $ helm install . --name my-release --set pfsd.replicaCount=3
+
+Alternatively, a YAML file that specifies the values for the above
+parameters can be provided while installing the chart. For example,
+
+.. code:: bash
+
+    $ helm install . --name my-release -f values.yaml
+
+.. Tip:: Use the default values.yaml file from the Zenko
+	 source at ~/Zenko/kubernetes/cosmos/values.yaml
+
+Installing the Chart with a Zenko Instance
+------------------------------------------
+ 
+To configure Cosmos on a Zenko instance, perform the following steps
+from `Orbit <https://admin.zenko.io>`_.
+ 
+1. Create a storage location of type "NFS Mount".
+ 
+2. Create a bucket within this location constraint.
+ 
+3. Create a user for Cosmos.
+ 
+4. Configure instance-specifc Cosmos values using information from
+   the previous steps.
+ 
+5. Export the following variables with appropiate values entered:
+ 
+   .. code:: bash
+
+      # Values from Orbit
+      export ACCESS_KEY=<your-cosmos-user-access-key>
+      export SECRET_KEY=<your-cosmos-user-secret-key>
+      export NFS_BUCKET=<your-cosmos-bucket-name>
+      export NFS_LOCATION=<your-nfs-mount-location-name>
+ 
+      # Values of your NFS mount point
+      export NFS_HOST=<your-nfs-server-host>
+      export NFS_EXPORT_PATH=<your-nfs-server-path>
+ 
+      # Cloudserver endpoint (assuming it's running on the same namespace)
+      export CLOUDSERVER_ENDPOINT=$(kubectl get svc -l app=cloudserver -o jsonpath='{.items[*].metadata.name}')
+ 
+6. Configure Cosmos.
```

Comment:
Create a proper cosmos configuration file

Reply:
Should there be a comparable configuration/export step when building from a freestanding cloudserver?